docs/zh-cn/README.md

```diff
@@ -1 +1,23 @@
 # GitHub 全域数字报告
+
+GitHub 数字全域报告是由 X-lab 发起的一个 GitHub 数字分析报告开源项目，这个项目旨在凝聚全球开发者的智慧共同对 GitHub 全域行为日志数据进行分析统计，以使开发者可以更好的理解和参与开源。
+
+## 数据
+
+我们使用 [GHArchive](https://www.gharchive.org/) 作为 GitHub 日志数据源，数据服务由 X-lab 维护的 [clickhouse](https://clickhouse.tech/) 集群提供. 关于数据的详细细节，请查看[数据](https://www.x-lab.info/github-analysis-report/#/data)文档。
+
+## 贡献指南
+
+在对项目贡献之前，请务必查阅我们的[贡献指南](https://www.x-lab.info/github-analysis-report/#/CONTRIBUTING)。
```

Comment:
I think it should be https://www.x-lab.info/github-analysis-report/#/zh-cn/CONTRIBUTING .

Reply:
Actually yes, but we still don't have this file.

Reply:
Sorry, we had one, I will fix this now.

Reply:
I will add it right now